### PR TITLE
Fix regression on marathon backend

### LINKED ISSUE #194: Marathon backend regression
Seems recent changes to the marathon template results in a single traefik backend getting created instead of a backend per app in marathon.

I'm suspecting that https://github.com/emilevauge/traefik/commit/2d79c500dfefaf71d282efeebdd58cf99cf8b474#diff-080894c4c7f765921235ddb2ceb82140 is a cause.

This is resulting in messages similar to:
```
time="2016-02-11T23:19:50Z" level=debug msg="Creating frontend frontend-registry-registry2"
time="2016-02-11T23:19:50Z" level=debug msg="Wiring frontend frontend-registry-registry2 to entryPoint http"
time="2016-02-11T23:19:50Z" level=debug msg="Creating route route-host-registry-registry2 Host:registry2.domain.local"
time="2016-02-11T23:19:50Z" level=debug msg="Creating backend backend-registry-registry2"
time="2016-02-11T23:19:50Z" level=error msg="Error loading new configuration, aborted Undefined backend: backend-registry-registry2"
```

An example of the generated config:
```
{
  "backends": {
    "backend-prod-service4": {
      "servers": {
        "server-prod-service4-026c4e04-d03e-11e5-b038-0242ac110002": {
          "url": "http://sc009.domain.local:31923"
        },
        "server-prod-service1-19072592-d039-11e5-b038-0242ac110002": {
          "url": "http://sc009.domain.local:31606"
        },
        "server-prod_service3-cceb9756-ca0f-11e5-8680-0242ac110002": {
          "url": "http://sc005.domain.local:31025"
        },
        "server-prod_service2-bb741921-cc61-11e5-a008-0242ac110002": {
          "url": "http://sc005.domain.local:31937"
        },
        "server-registry_registry-5b04fa67-c60f-11e5-8680-0242ac110002": {
          "url": "http://sc001.domain.local:31989"
        },
        "server-registry_registry2-5b04fa66-c60f-11e5-8680-0242ac110002": {
          "url": "http://sc002.domain.local:31920"
        },
        "server-registry_registryui-5b054888-c60f-11e5-8680-0242ac110002": {
          "url": "http://sc002.domain.local:31593"
        }
      }
    }
  },
  "frontends": {
    "frontend-prod-service4": {
      "backend": "backend-prod-service4",
      "routes": {
        "route-host-prod-service4": {
          "rule": "Host",
          "value": "prod-service4.domain.local"
        }
      },
      "passHostHeader": true
    },
    "frontend-prod-service3": {
      "backend": "backend-prod-service3",
      "routes": {
        "route-host-prod-service3": {
          "rule": "Host",
          "value": "prod-service3.domain.local"
        }
      },
      "passHostHeader": true
    },
    "frontend-prod-service2": {
      "backend": "backend-prod-service2",
      "routes": {
        "route-host-prod-service2": {
          "rule": "Host",
          "value": "prod-service2.domain.local"
        }
      },
      "passHostHeader": true
    },
    "frontend-prod-service1": {
      "backend": "backend-prod-service1",
      "routes": {
        "route-host-prod-service1": {
          "rule": "Host",
          "value": "prod-service1.domain.local"
        }
      },
      "passHostHeader": true
    },
    "frontend-registry-registry": {
      "backend": "backend-registry-registry",
      "routes": {
        "route-host-registry-registry": {
          "rule": "Host",
          "value": "registry.domain.local"
        }
      }
    },
    "frontend-registry-registry2": {
      "backend": "backend-registry-registry2",
      "routes": {
        "route-host-registry-registry2": {
          "rule": "Host",
          "value": "registry2.domain.local"
        }
      }
    },
    "frontend-registry-registryui": {
      "backend": "backend-registry-registryui",
      "routes": {
        "route-host-registry-registryui": {
          "rule": "Host",
          "value": "registryui.domain.local"
        }
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by emilevauge**
This PR fixes regression on marathon backend.
Fixes #194
cc @dontrebootme @vdemeester

### CODE REVIEW

File: tests/compose-marathon.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ slave:
     - /usr/bin/docker:/usr/bin/docker:ro
     - /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1:ro
     - /var/run/docker.sock:/var/run/docker.sock
+    - /lib/x86_64-linux-gnu/libsystemd-journal.so.0:/lib/x86_64-linux-gnu/libsystemd-journal.so.0
```
Comment (vdemeester) [inline tests/compose-marathon.yml]: wat ? :angel:

Comment (emilevauge) [inline tests/compose-marathon.yml]: This is needed with the last mesos version: https://github.com/mesosphere/docker-containers/issues/40

Comment (vdemeester) [inline tests/compose-marathon.yml]: Ah.. weird.. :sweat:

**Comment by vdemeester** [comment]
LGTM 🐏

**Pull request merged**